src/components/Header/index.jsx

```diff
@@ -97,8 +97,8 @@ function Header() {
   }
 
   return (
-    <div className="flex py-4 gap-10 lg:gap-20 justify-between items-center sticky top-0 z-10 bg-white ">
-      <div className="flex mr-auto ml-4 gap-4 items-center flex-shrink-0">
+    <div className="flex gap-10 lg:gap-20 justify-between items-center sticky top-0 z-10 bg-white ">
+      <div className="flex ml-4 mr-4 gap-4 items-center shrink">
```

Comment:
```suggestion
      <div className="flex mx-4 gap-4 items-center shrink">
```
이렇게 하나로 합칠 수 있을 것 같습니다!